docs/manual-EN/1.overview/2.quick-start/2.FAQ.md

```diff
@@ -12,7 +12,7 @@ Common questions about **Nebula Graph** and more.
     - [How to Check Runtime Configs](#how-to-check-runtime-configs)
     - [Connection Refused](#connection-refused)
     - [Process Crash](#process-crash)
-    - [Could not create logging file:... Too many open files](#could-not-create-logging-file-too-many-open-files)
+    - [`Could not create logging file:... Too many open files`](#could-not-create-logging-file-too-many-open-files)
```

Comment:
why add ``

Reply:
means this is the message returned by the console